src/wolfkrow/core/tasks/task.py

```diff
@@ -269,6 +298,22 @@ def __init__(self, **kwargs):
         if self.command_line_executable_args is None:
             self.command_line_executable_args = os.environ.get("WOLFKROW_DEFAULT_COMMAND_LINE_EXECUTABLE_ARGS")
 
+    @property
+    def task_name(self):
```

Comment:
This is an awful name. It conflicts too much with the `name` TaskAttribute. 

Lets change this to be `full_name` or something like that...